infra/dash/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3.8'
```

Comment:
to be revised by @Dragos0000

Reply:
compose version was chosen according to the versions used in ted sws